packages/nx/src/command-line/init/init-v2.ts

```diff
@@ -0,0 +1,171 @@
+import { existsSync } from 'fs';
+import { sync } from 'glob';
+import { PackageJson } from '../../utils/package-json';
+import { prerelease } from 'semver';
+import { output } from '../../utils/output';
+import { getPackageManagerCommand } from '../../utils/package-manager';
+import { generateDotNxSetup } from './implementation/dot-nx/add-nx-scripts';
+import { runNxSync } from '../../utils/child-process';
+import { readJsonFile } from '../../utils/fileutils';
+import { nxVersion } from '../../utils/versions';
+import {
+  addDepsToPackageJson,
+  askAboutNxCloud,
+  createNxJsonFile,
+  runInstall,
+  updateGitIgnore,
+} from './implementation/utils';
+import { prompt } from 'enquirer';
+import { execSync } from 'child_process';
+
+export interface InitArgs {
+  interactive: boolean;
+  nxCloud?: boolean;
+  useDotNxInstallation?: boolean;
+}
+
+export async function initHandler(options: InitArgs) {
```

Comment:
If this is run in an Angular CLI workspace, it wouldn't work because we don't have the plugin yet. This might also be the case with other plugins.

Should this handle that scenario and error? Or reroute it to the old handler?

Reply:
Added the custom Angular logic back for now.